cmd/oras/root/push.go

```diff
@@ -136,6 +143,9 @@ Example - Push file "hi.txt" into an OCI image layout folder 'layout-dir' with t
 				if opts.manifestConfigRef != "" && opts.artifactType != "" {
 					return errors.New("--artifact-type and --config cannot both be provided for 1.0 OCI image")
 				}
+				if opts.Platform.Platform != nil && opts.artifactType != "" {
+					return errors.New("--artifact-type and --platform cannot both be provided for 1.0 OCI image")
+				}
```

Comment:
They can be used together, can't they?

Reply:
Indeed they can be used together. We can populate the `artifactType` as `mediaType` into the config descriptor.

Reply:
Done.